ra/registration-authority_test.go

```diff
@@ -317,8 +317,8 @@ func TestValidateEmail(t *testing.T) {
 	}{
 		{"an email`", unparseableEmailDetail},
 		{"a@always.invalid", emptyDNSResponseDetail},
-		{"a@always.timeout", "DNS query timed out"},
-		{"a@always.error", "DNS networking error"},
+		{"a@always.timeout", "DNS query timed out during A-record lookup of always.timeout"},
```

Comment:
I'm confused why these would be A-record lookup failures, when `validateEmail` checks the MX-record (and sets `recQ := "MX"`) first.